pilot/pkg/proxy/envoy/v1/config.go

```diff
@@ -291,17 +292,18 @@ func buildSidecarListenersClusters(
 		httpOutbound = BuildExternalServiceHTTPRoutes(mesh, node, proxyInstances, config, httpOutbound)
 		clusters = append(clusters, httpOutbound.Clusters()...)
 		listeners = append(listeners, buildHTTPListener(buildHTTPListenerOpts{
-			mesh:             mesh,
-			proxy:            node,
-			proxyInstances:   proxyInstances,
-			routeConfig:      nil,
-			ip:               listenAddress,
-			port:             int(mesh.ProxyHttpPort),
-			rds:              RDSAll,
-			useRemoteAddress: useRemoteAddress,
-			direction:        traceOperation,
-			outboundListener: true,
-			store:            config,
+			mesh:                 mesh,
+			proxy:                node,
+			proxyInstances:       proxyInstances,
+			routeConfig:          nil,
+			ip:                   listenAddress,
+			port:                 int(mesh.ProxyHttpPort),
+			rds:                  RDSAll,
+			useRemoteAddress:     useRemoteAddress,
+			direction:            traceOperation,
+			outboundListener:     true,
+			store:                config,
+			authenticationPolicy: nil, /* authN policy is not needed for outbout listener */
```

Comment:
Then why we still explicitly set it to nil?

Reply:
Oh, this will make the code more readable, i think?

Reply:
Good catch :). I was adding those in order to config jwt filter for listener. But then decide to make PR smaller (also need to wait for proxy code in) by just config SSL in this PR. This is not needed, yet.